### PR TITLE
[WIP] fix failing CI

### PR DESCRIPTION
**Pull request by jhamman**
closes #80

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def create_dataset(
 
     ds.tmin.encoding['_FillValue'] = np.float32(-9999999)
     ds.tmax.encoding['_FillValue'] = np.float32(-9999999)
-    ds.time.encoding['bounds'] = 'time_bounds'
+    ds.time.attrs['bounds'] = 'time_bounds'
```
Comment (jhamman) [inline tests/utils.py:111]: @benbovy / @andersy005 - want to draw your attention here. This change fixed the failing test on main.

**Review by andersy005 — APPROVED** [review]

File: tests/utils.py (hunk 1 above)
Comment (andersy005) [reply tests/utils.py:111]: @jhamman, does this PR supersede PR #80?

Comment (jhamman) [reply tests/utils.py:111]: it does.

Comment (andersy005) [reply tests/utils.py:111]: My bad... I missed the comments in the PR body :)

**Pull request merged**